spec/unit/knife/bootstrap_windows_winrm_spec.rb

```diff
@@ -262,6 +284,7 @@
     let(:node_name) { 'foo.example.com' }
     before do
       allow(bootstrap).to receive(:wait_for_remote_response)
+      expect(bootstrap).to receive(:relay_winrm_command).with("echo %PROCESSOR_ARCHITECTURE%").and_return(arch_session_results)
```

Comment:
I always prefer expect to allow because it prevents bloat from forming in the specs over time as code is changed.  But if you would prefer, I will change it?